gapis/api/vulkan/frame_loop.go

```diff
@@ -527,6 +538,32 @@ func (f *frameLoop) buildStartEndStates(ctx context.Context, startState *api.Glo
 	f.loopStartState = f.cloneState(ctx, startState)
 	currentState := f.cloneState(ctx, startState)
 
+	st := GetState(currentState)
+	st.PreSubcommand = func(i interface{}) {
+		cr, ok := i.(CommandReferenceʳ)
+		if ok {
+			args := GetCommandArgs(ctx, cr, st)
+			switch ar := args.(type) {
+			case VkCmdBeginRenderPassArgsʳ:
+				rp := st.RenderPasses().Get(ar.RenderPass())
+				f.watcher.ignore = true
+				for i := uint32(0); i < uint32(rp.AttachmentDescriptions().Len()); i++ {
```

Comment:
I think there are more cases here that I missed.

IF the renderpass has LOAD_OP_CLEAR, then it really is a write (it will overwrite the renderpass)
possibly also if the renderpass has LOAD_OP_DONT_CARE (since that may overwrite the image as well)